src/NServiceBus.Core.Tests/Serializers/XML/Issue_2796.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace NServiceBus.Core.Tests.Serializers.XML
+{
+    using System;
+    using System.IO;
+    using NServiceBus.Serializers.XML.Test;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class Issue_2796
+    {
+        [Test]
+        public void Serialize_ShouldSucceed_WithAllNamespacesPresent()
```

Comment:
Arguably this test does not verify anything related to namespaces. i mean, it might be a good test, only the name might be off.